### DISCUSSION TITLE
Welcome to Duffy Discussions!

### DISCUSSION THREAD

**Discussion by patrickrills** [patrickrills/Duffy#77] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DrizzleBlitz** [comment]
Hello Patrick,
how can I play Duffy in French?
With my thanks

**Reply by patrickrills** [reply]
Some would need to translate these strings:
https://github.com/patrickrills/Duffy/blob/master/Duffy/en.lproj/Localizable.strings
